vars/buildPlugin.groovy

```diff
@@ -67,6 +67,12 @@ def call(Map addonParams = [:])
 	def deploy = addonParams.containsKey('deploy') && addonParams.deploy.metaClass.respondsTo('each') ? addonParams.deploy.findAll{ d -> d in platforms && d in PLATFORMS_DEPLOY && d in deployPlatforms } : PLATFORMS_DEPLOY
 	def version = addonParams.containsKey('version') && addonParams.version in VERSIONS_VALID ? addonParams.version : VERSIONS_VALID.keySet()[0]
 	def addon = env.JOB_NAME.tokenize('/')[1]
+	/**
+	 * Definition in case if an addon source code contains several addons,
+	 * then the start name of the various addons with Asterix is given with e.g.
+	 * `buildPlugin(various: 'USED_PREFIX.*', ...)`.
+	 */
+	def variousAddonsPackagePrefix = addonParams.containsKey('various') ? addonParams.various : null
```

Comment:
`various` as parameter name isn't descriptive. I would use something that contains archive(s) or artifact(s). Also assign the default value already here instead of doing it two time below.
```suggestion
	def archiveName = addonParams.containsKey('various') && addonParams.containsKey('various') != null ? addonParams.various : addon
```